package.json

```diff
@@ -65,6 +65,7 @@
     "@swc/core": "1.3.34",
     "@swc/jest": "0.2.24",
     "@types/jest": "27.0.2",
+    "@types/lodash": "4.14.191",
```

Comment:
This should be taken out from this PR, put in a refactor one

Reply:
Should be removed, I don't use it in the end. My mistake.